### PR TITLE
unindent gen ai metrics

### PR DESCRIPTION
**Pull request by bbqiu**
<details><summary>&#x1F6E0 DevTools &#x1F6E0</summary>
<p>

[Open in GitHub Codespaces](https://codespaces.new/bbqiu/mlflow/pull/10390?quickstart=1)

#### Install mlflow from this PR

```
pip install git+https://github.com/mlflow/mlflow.git@refs/pull/10390/merge
```

#### Checkout with GitHub CLI

```
gh pr checkout 10390
```

</p>
</details>

### Related Issues/PRs

 #xxx

### What changes are proposed in this pull request?
Before:
image
After:
image

### How is this PR tested?

- [ ] Existing unit/integration tests
- [ ] New unit/integration tests
- [ ] Manual tests

### Does this PR require documentation update?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. I've updated:
  - [ ] Examples
  - [ ] API references
  - [ ] Instructions

#### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

#### What component(s), interfaces, languages, and integrations does this PR affect?

Components

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/gateway`: AI Gateway service, Gateway client APIs, third-party Gateway integrations
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>

#### How should the PR be classified in the release notes? Choose one:

- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Review by liangz1 — APPROVED** [review]
LGTM!

File: docs/source/python_api/mlflow.metrics.rst [hunk 1]
```diff
@@ -68,16 +71,27 @@ We provide the following builtin factory functions to create :py:class:`Evaluati
...
 .. autofunction:: mlflow.metrics.f1_score
 
+Text Metrics
+---------------------
+
 .. autofunction:: mlflow.metrics.ari_grade_level
 
 .. autofunction:: mlflow.metrics.flesch_kincaid_grade_level
 
+Questions Answering Metrics
```
Comment (BenWilson2) [inline docs/source/python_api/mlflow.metrics.rst:90]:
```suggestion
Question Answering Metrics
```

File: docs/source/python_api/mlflow.metrics.rst [hunk 2]
```diff
@@ -68,16 +71,27 @@ We provide the following builtin factory functions to create :py:class:`Evaluati
...
+------------------------------------------
+
+Includes all of the above Text Metrics as well as the following:
```
Comment (BenWilson2) [inline docs/source/python_api/mlflow.metrics.rst:93]:
```suggestion
Includes all of the above **Text Metrics** as well as the following:
```

File: docs/source/python_api/mlflow.metrics.rst [hunk 3]
```diff
@@ -95,7 +109,7 @@ We provide the following builtin factory functions to create :py:class:`Evaluati
 .. autofunction:: mlflow.metrics.latency
 
 Retriever Metrics
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+---------------------
```
Comment (BenWilson2) [inline docs/source/python_api/mlflow.metrics.rst:112]:
```suggestion
-----------------
```

File: docs/source/python_api/mlflow.metrics.rst [hunk 4]
```diff
@@ -173,7 +187,7 @@ Users create their own :py:class:`EvaluationMetric <mlflow.metrics.EvaluationMet
     :exclude-members: MetricValue, EvaluationMetric, make_metric, EvaluationExample, ari_grade_level, flesch_kincaid_grade_level, exact_match, rouge1, rouge2, rougeL, rougeLsum, toxicity, answer_similarity, answer_correctness, faithfulness, answer_relevance, mae, mape, max_error, mse, rmse, r2_score, precision_score, recall_score, f1_score, token_count, latency, precision_at_k, recall_at_k, ndcg_at_k
 
 Generative AI Metrics
----------------------
+------------------------
```
Comment (BenWilson2) [inline docs/source/python_api/mlflow.metrics.rst:190]: Let's leave the alignment matching the text length.

```suggestion
---------------------
```

File: docs/source/python_api/mlflow.metrics.rst [hunk 5]
```diff
@@ -68,16 +71,27 @@ We provide the following builtin factory functions to create :py:class:`Evaluati
...
+---------------------
+
 .. autofunction:: mlflow.metrics.precision_score
 
 .. autofunction:: mlflow.metrics.recall_score
 
 .. autofunction:: mlflow.metrics.f1_score
 
+Text Metrics
+---------------------
```
Comment (BenWilson2) [inline docs/source/python_api/mlflow.metrics.rst:84]: Please match the H2 declaration to the text length of the preceding line.

File: docs/source/python_api/mlflow.metrics.rst [hunk 6]
```diff
@@ -68,16 +71,27 @@ We provide the following builtin factory functions to create :py:class:`Evaluati
...
+
 .. autofunction:: mlflow.metrics.ari_grade_level
 
 .. autofunction:: mlflow.metrics.flesch_kincaid_grade_level
 
+Questions Answering Metrics
+------------------------------------------
```
Comment (BenWilson2) [inline docs/source/python_api/mlflow.metrics.rst:91]: same here with length of H2 declaration

File: docs/source/python_api/mlflow.metrics.rst [hunk 7]
```diff
@@ -56,6 +56,9 @@ Evaluation results are stored as :py:class:`MetricValue <mlflow.metrics.MetricVa
 
 We provide the following builtin factory functions to create :py:class:`EvaluationMetric <mlflow.metrics.EvaluationMetric>` for evaluating models. These metrics are computed automatically depending on the ``model_type``. For more information on the ``model_type`` parameter, see :py:func:`mlflow.evaluate()` API.
 
+Regressor Metrics
+---------------------
```
Comment (BenWilson2) [inline docs/source/python_api/mlflow.metrics.rst:60]: H2 header declaration length should match the text of the header content

File: docs/source/python_api/mlflow.metrics.rst [hunk 8]
```diff
@@ -68,16 +71,27 @@ We provide the following builtin factory functions to create :py:class:`Evaluati
 
 .. autofunction:: mlflow.metrics.r2_score
 
+Classifier Metrics
+---------------------
```
Comment (BenWilson2) [inline docs/source/python_api/mlflow.metrics.rst:75]: Same here

**Review by BenWilson2 — APPROVED** [review]
LGTM! Thanks @bbqiu :)

**Pull request merged**